src/sphinx_book_theme/__init__.py

```diff
@@ -88,22 +88,36 @@ def hash_assets_for_files(assets: list, theme_static: Path, context):
     context: the Sphinx context object where asset links are stored. These are:
         `css_files` and `script_files` keys.
     """
-    for asset in assets:
+    for asset_path in assets:
         # CSS assets are stored in css_files, JS assets in script_files
-        asset_type = "css_files" if asset.endswith(".css") else "script_files"
+        asset_type = "css_files" if asset_path.endswith(".css") else "script_files"
         if asset_type in context:
             # Define paths to the original asset file, and its linked file in Sphinx
-            asset_sphinx_link = f"_static/{asset}"
-            asset_source_path = theme_static / asset
+            asset_sphinx_link = f"_static/{asset_path}"
+            asset_source_path = theme_static / asset_path
             if not asset_source_path.exists():
                 SPHINX_LOGGER.warning(
                     f"Asset {asset_source_path} does not exist, not linking."
                 )
             # Find this asset in context, and update it to include the digest
-            if asset_sphinx_link in context[asset_type]:
-                hash = _gen_hash(asset_source_path)
-                ix = context[asset_type].index(asset_sphinx_link)
-                context[asset_type][ix] = asset_sphinx_link + "?digest=" + hash
+            for i, other_asset in enumerate(context[asset_type]):
```

Comment:
nitpick but I'm a fan of `ii` instead of `i` so that it's easier to ctrl+f if need be 🤷 . Disregard this if you wish!

Reply:
Fine, no strong feelings!